simulation/traffic_simulator/include/traffic_simulator/entity/entity_manager.hpp

```diff
@@ -246,6 +247,10 @@ class EntityManager
 
   void setTargetSpeed(const std::string & name, double target_speed, bool continuous);
 
+  void requestSpeedChange(
+    const std::string & name, const double target_speed, const SpeedChangeTransition transition,
+    const SpeedChangeConstraint constaint, const bool continuous);
```

Comment:
`constaint` => `constraint`

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/648/commits/b608d5e2c700eea9af558ed92a2a6d2699994108